Tensile/KernelWriter.py

```diff
@@ -1546,7 +1544,9 @@ def kernelBody( self, kernel, tensorParametersA, tensorParametersB ):
         if self.enable["LocalRead"]:
           for plrIdx in range(0, self.numItersPLR):
             pack[plrIdx] = Code.Module()
-            for espi in range(0, (self.prefetchAcrossPersistent and kernel["ExpandPointerSwap"])+1):
+            # no matter EPS or PAP, only prefect local once per plrIdx
+            # for espi in range(0, (self.prefetchAcrossPersistent and kernel["ExpandPointerSwap"])+1):
+            for espi in range(0, 1):
```

Comment:
Does this alter the behavior of EPS?

Reply:
Actually not, since prefetchAcrossPersistent + EPS was not supported before, so this "and" was always evaluated as False (0). Thus the range(0,+1) = range(0,1),